### DISCUSSION TITLE
Mac Version

### DISCUSSION THREAD

**Discussion by CodingMax** [software-2/LightroomSync#3] [General]
Hey, i want to sync my mac library with the one on my win desktop, so i would like to try to make a mac version of the tool.
Can you maybe guide me a bit?
I learned some swift and know some c sharp and will also be able to learn anything else needed. Would just need some guidance on where to start :) And maybe someone chimes in at some point.

**Comment by software-2** [comment]
The sync files will need to match the same JSON output as the Windows program, and the zip files will need to follow the same naming convention. As long as those things happen, it should be compatible.

Things to look out for:
Make sure there's a lockstep to prevent lightroom from being open multiple places at once.
Keep old versions of zips, deleting those is left to the user intentionally.

If you have questions, feel free to ask. I'm on the road right now, so I may be slow to respond.
